quickstarts/library-process/library-client/src/main/java/org/openshift/quickstarts/rhpam/kieserver/library/client/LibraryClient.java

```diff
@@ -303,31 +303,38 @@ boolean returnLoan(Loan loan) {
 
     // only needed for non-production test scenarios where the TLS certificate isn't set up properly
     private void forgiveUnknownCert() throws Exception {
-        KieRemoteHttpRequest.ConnectionFactory connf = new KieRemoteHttpRequest.ConnectionFactory() {
+        KieServerHttpRequest.ConnectionFactory connf = new KieServerHttpRequest.ConnectionFactory() {
+
             public HttpURLConnection create(URL u) throws IOException {
-                return forgiveUnknownCert((HttpURLConnection)u.openConnection());
+                return forgiveUnknownCert((HttpURLConnection) u.openConnection());
             }
+
             public HttpURLConnection create(URL u, Proxy p) throws IOException {
-                return forgiveUnknownCert((HttpURLConnection)u.openConnection(p));
+                return forgiveUnknownCert((HttpURLConnection) u.openConnection(p));
             }
+
             private HttpURLConnection forgiveUnknownCert(HttpURLConnection conn) throws IOException {
                 if (conn instanceof HttpsURLConnection) {
                     HttpsURLConnection sconn = HttpsURLConnection.class.cast(conn);
                     sconn.setHostnameVerifier(new HostnameVerifier() {
+
                         public boolean verify(String arg0, SSLSession arg1) {
                             return true;
                         }
                     });
                     try {
                         SSLContext context = SSLContext.getInstance("TLS");
-                        context.init(null, new TrustManager[] {
-                            new X509TrustManager() {
-                                public void checkClientTrusted(X509Certificate[] chain, String authType) throws CertificateException {}
-                                public void checkServerTrusted(X509Certificate[] chain, String authType) throws CertificateException {}
-                                public X509Certificate[] getAcceptedIssuers() {
-                                    return null;
-                                }
-                            }
+                        context.init(null, new TrustManager[]{
+                                                              new X509TrustManager() {
+
```

Comment:
Why introduce such a huge indent? Are you using spaces (preferred) or tabs?

Reply:
I'm using spaces and this is a consequence of the droolsjbpm formatter profile. I can revert back to the original one, but will break the formatter profile.

Reply:
Nah, it's fine. It's that big of a deal to me. :)